### PR TITLE
SEC-79: Install nginx from their ppa

### PR DESCRIPTION
**Pull request by maxrothman**
This change allows us to install nginx from nginx's ppa as per their
instructions at http://nginx.org/en/linux_packages.html

@edx/devops

### CODE REVIEW

File: playbooks/roles/nginx/tasks/main.yml [hunk 1]
```diff
@@ -33,8 +33,37 @@
...
+    repo: deb http://nginx.org/packages/ubuntu/ {{ ansible_distribution_release }} nginx
+    state: present
+  notify: restart nginx
+  tags:
+    - install
+    - install:system-requirements
+
+- name: Install the nginx package
+  apt:
+    name: nginx
```
Comment (feanil) [inline playbooks/roles/nginx/tasks/main.yml:78]: This will auto-upgrade NGINX, I think we should probably pin instead.

Comment (maxrothman) [inline playbooks/roles/nginx/tasks/main.yml:78]: Unfortunately, nginx doesn't distribute multiple versions, just a "stable" and a "mainline", as you can see from their [documentation](http://nginx.org/en/linux_packages.html) and their [repo](http://nginx.org/packages/ubuntu/).

Comment (e0d) [inline playbooks/roles/nginx/tasks/main.yml:78]: I think I'd like to repo defined in `defaults/main.yml` and uppercase.  I see a couple cases where we would like to override.
- where we host our own package.
- where we need a customized package like that provided by AppNeta.

**Comment by fredsmith** [comment]
:+1:

**Comment by jibsheet** [comment]
:+1:

**Pull request merged**